{
  "name": "try-json-parse",
  "version": "0.1.1",
  "description": "Doesn't throw an error when JSON.parse() fails, just returns undefined",
  "main": "index.js",
  "scripts": {
    "test": "node test.js"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/alessioalex/try-json-parse"
  },
  "keywords": [
    "err",
    "custom",
    "custom-error",
    "customerror"
  ],
  "author": {
    "name": "Alexandru Vladutu",
    "email": "alexandru.vladutu@gmail.com",
    "url": "http://careers.stackoverflow.com/alessioalex"
  },
  "license": "MIT",
  "bugs": {
    "url": "https://github.com/alessioalex/try-json-parse/issues"
  },
  "readme": "### try-json-parse\r\n\r\nJust like `JSON.parse()`, except it returns `undefined` instead of throwing an error for invalid JSON.\r\n\r\n### Usage\r\n\r\n```js\r\nvar JSONparse = require('./index'),\r\n    assert    = require('assert'),\r\n    fs        = require('fs');\r\n\r\nfs.readFile(__dirname + '/package.json', function(err, content) {\r\n  var pkgJson;\r\n\r\n  if (err) { throw err; }\r\n\r\n  pkgJson = JSONparse(content);\r\n  assert.equal(pkgJson.name, 'try-json-parse');\r\n  assert(JSONparse(fs.readFileSync(__filename)) === undefined);\r\n});\r\n```\r\n\r\n### Motivation\r\n\r\nGot tired of writing try {} catch {}.\r\n\r\n### License\r\n\r\nMIT\r\n",
  "readmeFilename": "README.md",
  "_id": "try-json-parse@0.1.1",
  "_from": "try-json-parse@^0.1.1"
}
